etna/metrics/metrics.py

```diff
@@ -149,7 +155,8 @@ def __init__(self, mode: str = MetricAggregationMode.per_segment, **kwargs):
         kwargs:
             metric's computation arguments
         """
-        super().__init__(mode=mode, metric_fn=mape, **kwargs)
+        mape_per_output = partial(mape, mode="per_output")
```

Comment:
Don't we want to make naming consistent with sklearn? mode -> multioutput?

Reply:
Ok, but we should probably also discuss selected names for `mode`.